bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -308,6 +308,7 @@ impl<'a> CircuitInputStateRef<'a> {
         if (!matches!(op.field, AccountField::CodeHash)
             && (matches!(rw, RW::READ) || (op.value_prev.is_zero() && op.value.is_zero())))
             && account.is_empty()
+            && !self.tx.invalid_tx
```

Comment:
Why would this be okay to do this for invalid transactions? Seems like we still have the same limitations in this case where we have to read state from the MPT which has this limitation. I will also look into how to best remove this tomorrow because this could be tricky.

Reply:
Without this line of code change, it returns
```
thread 'evm_circuit::execution::begin_tx::test::begin_tx_enable_skipping_invalid_tx' panicked at 'RWTable Account field WRITE lookup to non-existing account rwc: 14, op: AccountOp { addr: 0x0000000000000000000000000000000000000000, field: Balance, val_prev: 0x0, val: 0x0 }', bus-mapping/src/circuit_input_builder/input_state_ref.rs:312:13
```

Reply:
Yes, but it's a good check! Because even with invalid transactions we could be reading data from accounts that don't exist, and we have to actually do that read. And so whatever we do, the circuits have to be able to support it otherwise we cannot prove it. This check says that what we're doing is not supported so we cannot prove it.

Will look into this later today.

Reply:
This problem isn't related to the invalid transactions directly, it's a problem with all blocks that do not pay any fee to the coinbase address. Will make an issue/PR on the PSE repo.